internal/pkg/cli/task_run.go

```diff
@@ -141,8 +219,8 @@ func BuildTaskRunCmd() *cobra.Command {
 		Example: `
 Run a task with default setting.
 /code $ copilot task run
-Run a task in the "test" environment under the current workspace.
-/code $ copilot task run --env test
+Run a task named "db-migrate" in the "test" environment under the current workspace.
+/code $ copilot task run -n db-migrate --env test
```

Comment:
We might want to rethink about using -n as the flag (to me -n makes more sense for num) Would it make sense to name it family-name instead?

Reply:
That's true...My concern was that (assuming we want to have a short form for this flag because it is a required flag), if we use `--task-name` or `--task-family`, then it would probably make more sense to have `-t` as the short form. However, in app, svc, env, etc. it is always `-n`. Would it be considered inconsistent?

Reply:
Ah, makes sense to keep it -n for consistency, but I think --task-name (--run-name or something along those lines) adds some clarity (that the task will be named as such, as well as the stack and all of it's resources?). Thoughts?

Reply:
Ahh yes I agree. --name sounds like it is the name for a single task, instead of a group of similar tasks as intended. Probably something involving 'group' or 'family' might be good in order to emphasize that? --task-family-name as the long form and -n as the short form?

Reply:
Actually `svc init` previously has the name flag called `--svc` instead of `--name`. And the reason why we made this change is because it seems to be redundant to have` svc init --svc frontend`, since `svc init --name frontend` already has enough background to know what `--name` refers to. I think maybe the same rule could be applied to `task run --name db-migrate` as well?

Reply:
I agree with using `--task-family-name/-n` for the name flag. We can use `--count / -c` for the number of tasks to run.

Reply:
--count / -c makes sense to me

Reply:
I agree with using `--count / -c`!

I also agree with @pkandasamy91 on that `--task-family-name` could be potentially confusing. The ECS task definition has a `family` property, while we don't mean exactly the same thing here since it also involves cw log group, ecr repo, etc. 

I am also proposing `--task-group-name` because it not only indicates that multiple tasks can be "grouped" together, but also is different than the "family" property in ECS task definition.